net/nimble/host/mesh/src/net.c

```diff
@@ -556,40 +556,57 @@ void bt_mesh_iv_update(u32_t iv_index, bool iv_update)
 {
 	int i;
 
-	if (iv_index < bt_mesh.iv_index || iv_index > bt_mesh.iv_index + 42) {
-		BT_ERR("IV Index completely out of sync: 0x%08x != 0x%08x",
-			iv_index, bt_mesh.iv_index);
-		return;
-	}
+	/* If we are in normal*/
+	if (!bt_mesh.iv_update) {
 
```

Comment:
I think you should add an initial check for (iv_index == bt_mesh.iv_index) here, since otherwise you'll get confusing errors for perfectly normal Secure Network Beacons which have the same IV Index as you.

Reply:
Which errors are you referring to?